src/main/java/frc/utils/applicationsutils/CMDHandler.java

```diff
@@ -21,17 +21,21 @@ public static void runCMDCommand(Path directory, String command) {
     }
 
     public static void runCMDCommand(String command) {
-        String cmdSpecification = isWindows() ? WINDOWS_CMD_SPECIFICATION : NON_WINDOWS_CMD_SPECIFICATION;
+        String operatingSystemShell = isWindows() ? WINDOWS_SHELL : NON_WINDOWS_SHELL;
+
+        FileCreator.writeToOutputFile(OUTPUT_FILE, "\nRunning: " + command, true);
```

Comment:
Suggested change
```suggestion
        FileCreator.writeToOutputFile(OUTPUT_FILE, "\nTrying to run: " operatingSystemShell + command, true);
```
I want the full line to be easier to debug, also i think this line should be in try right before the runtime.exec